js/gated-content/src/views/HomeWrapper.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div v-if="!config.switch_legacy_view">
```

Comment:
Probably you wanted to use showLegacyComponents here? :)

Reply:
I think it's just better to access the config directly here, instead of using simple wrapper method, as it is a boolean only. So i forgot to remove the method :) What do you think?

Reply:
Both options are fine for me :)